superdesk/publish/publish_content.py

```diff
@@ -96,54 +96,76 @@ def get_queue_items(retries=False):
     return get_resource_service(PUBLISH_QUEUE).get(req=request, lookup=lookup)
 
 
-@celery.task(soft_time_limit=1800, bind=True)
+@celery.task(soft_time_limit=60, bind=True)
 def transmit_subscriber_items(self, queue_items, subscriber):
     # Attempt to obtain a lock for transmissions to the subscriber
     lock_name = get_lock_id("Subscriber", "Transmit", subscriber)
     if not lock(lock_name, expire=300):
```

Comment:
imo the original problem could be this expire value which is lower than soft_time_limit, so if it gets stuck for 30m it will be able to start this task again every 5m and thus block more workers